nightly-5.3/centos/8/Dockerfile

```
@@ -0,0 +1,70 @@
+FROM centos:8
+LABEL maintainer="Swift Infrastructure <swift-infrastructure@swift.org>"
+LABEL description="Docker Container for the Swift programming language"
+
+RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+
+RUN yum -y update && yum install --enablerepo=PowerTools -y \
```

```suggestion
RUN yum install --enablerepo=PowerTools -y \
```
Can you drop `yum -y update` from all Dockerfiles?

---

@shahmishal Branch updated and `yum -y update` removed from all.